### PR TITLE
New onboarding with notification

### PR DESCRIPTION
**Pull request by PanRadaCloudbest**
Only Screens with simple sagas logic and connector

### CODE REVIEW

File: loc/en.js [hunk 1]
```diff
@@ -238,9 +241,14 @@ module.exports = {
     changePin: 'Change PIN',
     confirmNewPin: 'Confirm new PIN',
     confirmPassword: 'Confirm transaction password',
+    confirmNotification: 'Confirm',
     confirmPin: 'Confirm PIN',
     createNewPin: 'New PIN',
+    confirmEmail: 'Confirm your email address',
+    confirmEmailDescription: 'Please enter the code that we sent to : verylongname@email.com',
```
Comment (marek-siemieniuk-morawski) [inline loc/en.js:248]: do we want to have email address hardcoded here?

File: src/screens/AddNotificationEmailScreen/AddNotificationEmailScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+  };
+
+  setEmail = (email: string): void => {
+    this.setState({
+      email,
+    });
+  };
+
+  onSave = () => {
+    //TODO: pass email and connect with api
```
Comment (marek-siemieniuk-morawski) [inline src/screens/AddNotificationEmailScreen/AddNotificationEmailScreen.tsx:41]: shouldn't be here data validation on email address? 🤔

File: src/screens/AddNotificationEmailScreen/AddNotificationEmailScreen.tsx [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+    this.props.navigation.navigate(Route.ConfirmNotificationCode);
+  };
+
+  skipAddEmail = () => {
+    const { createTc, createNotificationEmail, navigation } = this.props;
+    createTc();
```
Comment (marek-siemieniuk-morawski) [inline src/screens/AddNotificationEmailScreen/AddNotificationEmailScreen.tsx:47]: as far as I understand, `createTc()` redirects user to Terms & Conditions screen. If I'm right then the flow seems incorrect. It should be Terms & Conditions > Onboading > AddNotificationEmailScreen, right?

File: src/screens/AddNotificationEmailScreen/AddNotificationEmailScreen.tsx [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+        footer={
+          <>
+            <Button
+              title={i18n.onboarding.confirmNotification}
+              testID="submit-notification-email"
+              onPress={this.onSave}
+              disabled={email.length === 0}
+            />
+            <FlatButton
+              testID="skip-notification-email"
```
Comment (marek-siemieniuk-morawski) [inline src/screens/AddNotificationEmailScreen/AddNotificationEmailScreen.tsx:80]: I like it

**Review by marek-siemieniuk-morawski — APPROVED** [review]

**Comment by marek-siemieniuk-morawski** [comment]
typo in PR name - it should be "notification"

File: src/screens/AddNotificationEmailScreen/AddNotificationEmailScreen.tsx (hunk 3 above)
Comment (PanRadaCloudbest) [reply src/screens/AddNotificationEmailScreen/AddNotificationEmailScreen.tsx:47]: createTc() is for add info this user finish and accepted TC

File: src/screens/AddNotificationEmailScreen/AddNotificationEmailScreen.tsx (hunk 4 above)
Comment (PanRadaCloudbest) [reply src/screens/AddNotificationEmailScreen/AddNotificationEmailScreen.tsx:80]: me too :)

File: src/screens/AddNotificationEmailScreen/AddNotificationEmailScreen.tsx (hunk 3 above)
Comment (marek-siemieniuk-morawski) [reply src/screens/AddNotificationEmailScreen/AddNotificationEmailScreen.tsx:47]: Got it, nvm then.

**Pull request merged**